style.css

```diff
@@ -97,16 +107,14 @@ header {
 .skills-container {
```

Comment:
- [ ] [OPTIONAL] You can align this items to the center, it will looks more like the original design

Reply:
I literally thought I messed it up! Thanks for the feedback. 

I actually wanted to align the skills container but it wasn't aligned in the design when I took a look at it but I think it looks better in the centre so I'll do that!